CHANGES.md

```diff
@@ -1,5 +1,67 @@
 # Changelog
 
+# 1.5.10 / 0.21.10 (libraries)
+
+### New
+
+- [asset backfills] Previously, when partitions definitions were changed after backfill launch, the asset backfill page would be blank. Now, when partitions definitions are changed, the backfill page will display statuses by asset.
+- Added a new `MetadataValue.job` metadata type, which can be used to link to a Dagster job from other objects in the UI.
+- [dagster-bigquery, dagster-duckdb, dagster-snowflake]. The BigQuery, DuckDB, and Snowflake I/O Managers will now determine the schema (dataset for BigQuery) in the following order of precedence: `schema` metadata set on the `asset` or `op`, I/O manager `schema`/ `dataset` configuration, `key_prefix` set on the `asset`. Previously, all methods for setting the schema/dataset were mutually exclusive, and setting more than one would raise an exception.
+- [dagster-shell] Added option to exclude the shell command from logs
+- [dagster-dbt] When running `DAGSTER_DBT_PARSE_PROJECT_ON_LOAD=1 dagster dev` in a new scaffolded project from `dagster-dbt project scaffold`, dbt artifacts for loading the project are now created in a static `target/` directory.
+
+### Bugfixes
+
+- Problematic inheritance that was causing pydantic warnings to be emitted has been corrected.
+- Its now possible to use the logger of `ScheduleEvaluationContext` when testing via `build_schedule_context`
+- The `metadata` from a `Failure` exception is now hoisted up to the failure that culminates when retry limits are exceeded.
+- Fixed bug in which the second instance of an hour partition at a DST boundary would never be shown as “materialized” in certain UI views.
+- [auto-materialize] Fix issue where assets which were skipped because required parent partitions did not exist would not be materialized once those partitions came into existence.
+- [UI] Fixed an issue where the test sensor dialog for a sensor that targeted multiple jobs would claim that all of the runs were targeting the same job.
+- [dagster ecs] The exit code of failed containers is now included in the failure message.
+- [dagster pipes] The `PipesK8sClient` now correctly raises on failed containers.
+- [dagster pipes] Using pipes within ops instead of assets no longer enforces problematic constraints.
+- [helm] Added `maxCatchupRuns` and `maxTickRetries` configuration options for the scheduler in the Helm chart.
+- Fixed an issue where backfilling an hourly partition that occurred during a fall Daylight Savings Time transition sometimes raised an error.
+- [embedded-elt] Fixed crashes for non-unicode logs
```

Comment:
period and move UI sections together
```suggestion
- [dagster ecs] The exit code of failed containers is now included in the failure message.
- [dagster pipes] The `PipesK8sClient` now correctly raises on failed containers.
- [dagster pipes] Using pipes within ops instead of assets no longer enforces problematic constraints.
- [helm] Added `maxCatchupRuns` and `maxTickRetries` configuration options for the scheduler in the Helm chart.
- Fixed an issue where backfilling an hourly partition that occurred during a fall Daylight Savings Time transition sometimes raised an error.
- [embedded-elt] Fixed crashes for non-unicode logs.
- [UI] Fixed an issue where the test sensor dialog for a sensor that targeted multiple jobs would claim that all of the runs were targeting the same job.
```